entity-framework/core/providers/in-memory/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: InMemory Database Provider - EF Core
+description: Information on the Entity Framework Core InMemory database provider
```

Comment:
Not sure if we are but we should be consistent about how we stylized inmemory in docs.

Reply:
Agreed, but for now I'm just reusing the same styling used in the title... Let's scope out of this already-annoying PR...

Reply:
Sure. Just file an issue.

Reply:
#2602